Add Composure Guide

JakeGWater opened the pull request:
- [x] hint admonition should be blue
- [x] link to lighting section 
- [x] mark time code and genlock section as planned
- [ ] Link to sdi converter
- [ ] Add deck link settings help
- [x] Green link to vive guide 
- [x] Garbage matte section
- [ ] Preview transform
- [x] Add equipment comparability guide

JakeGWater committed:
composure

JakeGWater committed:
checkin

JakeGWater committed:
comp

JakeGWater committed:
Merge branch 'main' of https://github.com/JakeGWater/vpifg.com into composure

JakeGWater committed:
Click to Expand Images

JakeGWater committed:
save

JakeGWater committed:
checkin

JakeGWater committed:
save

JakeGWater committed:
Merge branch 'main' into composure

JakeGWater committed:
Update unreal-composure.rst

JakeGWater committed:
Merge branch 'main' into composure

JakeGWater committed:
Merge branch 'main' into composure

JakeGWater committed:
save

JakeGWater committed:
figure directive

JakeGWater committed:
save

JakeGWater committed:
save

NLGRSTN committed:
Update index.rst

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/JakeGWater/vpifg.com?pullRequest=27) <br/>All committers have signed the CLA.

JakeGWater committed:
save

JakeGWater committed:
save

JakeGWater committed:
save

JakeGWater committed:
freeze

JakeGWater committed:
Merge branch 'main' of https://github.com/JakeGWater/vpifg.com into composure

JakeGWater committed:
pulling fix from upstraem

https://github.com/pydata/pydata-sphinx-theme/commit/3455c23ad887401eae585c37921178faf99ef4e5

JakeGWater committed:
save
